### PR TITLE
OEUI-284: Fetch Orders after discontinue and remove sinon.spy usage

### PR DESCRIPTION
**Pull request by Daramola98**
### **JIRA TICKET NAME**
[OEUI-284 Fetch Orders after discontinue and remove sinon.spy usage](https://issues.openmrs.org/browse/OEUI-284) 

### **Summary**
- The fetch orders action should be called after an order has been discontinued.
- The usage of Sinon.spy should be deprecated from the codebase.

## I have checked that on this Pull request

- [x] I can sucessfully create Drug orders
- [x] I can sucessfully create Lab orders
- [x] I can sucessfully search for drug orders

### CODE REVIEW

**Comment by dkayiwa** [comment]
@Daramola98 is this ready for review?

**Comment by Daramola98** [comment]
screen shot 2018-10-05 at 3 23 31 pm
screen shot 2018-10-05 at 3 24 52 pm

File: tests/actions/patient.test.js [hunk 1]
```diff
@@ -50,4 +65,4 @@ describe('Patient actions', () => {
       });
     done();
   });
-});
+});
```
Comment (emmabaye) [inline tests/actions/patient.test.js]: Good job. However, there should be a new line at the end of this file.

File: tests/reducers/patientReducer.test.js [hunk 2]
```diff
@@ -27,4 +39,4 @@ describe('Patient Reducer', () => {
       const newState = patientReducer(undefined, action);
       expect(newState).toEqual({ patient });
     });
-});
+});
```
Comment (emmabaye) [inline tests/reducers/patientReducer.test.js]: There should be a new line at the end of this file.

**Comment by Daramola98** [comment]
Done

**Comment by muhozi** [comment]
@Daramola98 Good job. Please work on the tests, the coverage has decreased.

**Comment by Daramola98** [comment]
> @Daramola98 Good job. Please work on the tests, the coverage has decreased.

The reason was stated above

**Pull request merged**